ansible/roles/openregister_config/templates/openregister-config.yaml.j2

```diff
@@ -58,6 +58,10 @@ downloadConfigs: true
 
 server:
   registerDefaultExceptionMappers: false
+  requestLog:
+    type: classic
```

Comment:
hmmm, I _think_ it's related to the format that logback uses but I'll double check and see what it's actually doing.